requirements.txt

```diff
@@ -52,7 +52,6 @@ google-auth-oauthlib==0.4.6
 google-pasta==0.2.0
 googleapis-common-protos==1.56.2
 gpustat==0.6.0
-grpcio==1.32.0
```

Comment:
No need to make changes to `SMARTS/requirements.txt` as this will be overwritten.